documentation/polymer-templates/tutorial-template-mapped-components-limitations.asciidoc

```diff
@@ -62,14 +56,63 @@ To summarize:
 * You can still use mutation API methods, like `appendChild`, `setProperty` or `setAttribute` from the server side, without issue.
 * Getter methods return values that are set from the server side only.
 
+== Enabled State
+
+Since hierarchical structure in the template is not reflected on the server side
+the `setEnabled` method called on `@Id` injected component doesn't disable any other 
+`@Id` injected component even though such component may be a descendant in the shadow DOM
+(see also <<../components/tutorial-enabled-state#, Component Enabled State>>).
+
+*Example*: Disable `@Id` injected component.
+
+[source,java]
+----
+@Tag("main-layout")
+@JsModule("./com/example/main-layout.js")
+public class MainLayout extends LitTemplate {
+
+    @Id("layout")
+    private VerticalLayout layout;
+    @Id("textfield")
+    private TextField textField;
+    @Id("button")
+    private Button button;
+
+    public void onSomeAction() {
+        layout.setEnabled(false);
+
+        System.out.println(textField.isEnabled()); // prints "true"
+        System.out.println(button.isEnabled()); // prints "true"
+    }
+}
+----
+
+[source,js]
+----
+class MainLayout extends LitElement {
+
+    render() {
+        return html`
+           <div>
+             <vaadin-vertical-layout id="layout">
+               <vaadin-text-field id="textfield"></vaadin-text-field>
+               <vaadin-button id="button"></vaadin-button>
+             </vaadin-vertical-layout>
+            </div>`;
+    }
+}
+
+customElements.define("main-layout", MainLayout);
+----
+
```

Comment:
The example shows the issue but it could also be good to hint at the fix. E.g., "`setEnabled(false)` must be called explicitly on the `textfield` and `button` components if disabling them is desired".